core-bundle/contao/themes/flexible/styles/main.css

```diff
@@ -2558,14 +2563,6 @@ time[title] {
 	cursor: help;
 }
 
-.float_left {
```

Comment:
Are we sure that nobody ever used these classes outside of `.cte_preview`?

Reply:
Hm, good point. So leave the original as is and just add `.cte_preview .media--left figure` (etc.) then?